homeassistant/components/soundtouch/media_player.py

```diff
@@ -78,32 +78,31 @@ class SoundTouchMediaPlayer(MediaPlayerEntity):
     _attr_device_class = MediaPlayerDeviceClass.SPEAKER
     _attr_has_entity_name = True
     _attr_name = None
+    _attr_source_list = [
+        Source.AUX.value,
+        Source.BLUETOOTH.value,
+    ]
 
     def __init__(self, device: SoundTouchDevice) -> None:
         """Create SoundTouch media player entity."""
 
         self._device = device
 
-        self._attr_unique_id = self._device.config.device_id
+        self._attr_unique_id = device.config.device_id
         self._attr_device_info = DeviceInfo(
-            identifiers={(DOMAIN, self._device.config.device_id)},
+            identifiers={(DOMAIN, device.config.device_id)},
             connections={
-                (CONNECTION_NETWORK_MAC, format_mac(self._device.config.mac_address))
+                (CONNECTION_NETWORK_MAC, format_mac(device.config.mac_address))
             },
             manufacturer="Bose Corporation",
-            model=self._device.config.type,
-            name=self._device.config.name,
+            model=device.config.type,
+            name=device.config.name,
         )
 
         self._status = None
         self._volume = None
         self._zone = None
 
-    @property
```

Comment:
removing this is causing issues because the "slaves" passed to things like "add_zone_slave" come from services using entity selector.

Reply:
Good catch, now that I open the changed files I see that there were indeed test failures. Strange I did not count them here.

Reply:
Oh nvm, the tests were failing but didnt see it